packages/server/modules/workspaces/tests/integration/workspaces.graph.spec.ts

```diff
@@ -33,7 +37,57 @@ import {
 import { BasicTestCommit, createTestCommit } from '@/test/speckle-helpers/commitHelper'
 import { BasicTestStream, createTestStream } from '@/test/speckle-helpers/streamHelper'
 import knex from '@/db/knex'
+import { createRandomPassword } from '@/modules/core/helpers/testHelpers'
+import { getBranchesByStreamId } from '@/modules/core/services/branches'
+
+const createProjectWithVersions =
```

Comment:
I created this utility simulating the flow to create versions of a model in a project, I would like to make sure the flow is correct
@gjedlicska